src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -12764,8 +12763,8 @@ private void UnifyUnderlyingWithTuple_04_AssertCompilation(CSharpCompilation com
                          m1Tuple.ToTestDisplayString());
         }
 
-        [Fact]
-        public void UnifyUnderlyingWithTuple_05()
+        [ConditionalFact(typeof(DesktopOnly))]
+        public void UnifyUnderlyingWithTuple_05k()
```

Comment:
Is the `k` intentional here?

Reply:
No. It's a vim slip up :( 

---
In reply to: [171427919](https://github.com/dotnet/roslyn/pull/25050#discussion_r171427919) [](ancestors = 171427919)